src/bot/botutils.jl

```diff
@@ -0,0 +1,19 @@
+################################################################
+import Pkg
+"""
+Should be removed once Pkg allows adding test dependencies to the current environment
+
+Used in Github Action workflow yaml file
+"""
+function addtestdep()
+    local testToml
+    try
+        testToml = Pkg.Types.parse_toml("test/Test.toml")
+    catch
+        error("please add a Test.toml to the /test directory for test dependencies")
```

Comment:
This seems like it is introducing a hole new way to give test dependencies. What is the use case?

Also, instead of a try catch, it seems that a `isfile` would work just as well considering the error message.